### DISCUSSION TITLE
Install Issues

### DISCUSSION THREAD

**Discussion by leighericksen** [PeterH24x7/-Solar-Analytics-integration-for-Home-Assistant-Energy-monitoring#8] [Q&A] [answered]
Hello, 

Was so relieved to find this integration for HASS but sadly having issues installing.

I'm currently getting the error:

"The system cannot restart because the configuration is not valid: Integration error: packages - Integration 'packages' not found."

When I go to restart my instance of HASS. Have followed the instructions as supplied but no luck.Have attached the configuration.yaml as it stands below:

`
# Loads default set of integrations. Do not remove.
default_config:

packages:
         pack_1: !include solar_analytics.yaml

# Text to speech
tts:
  - platform: google_translate

frontend:
  themes: !include_dir_merge_named themes
automation: !include automations.yaml
script: !include scripts.yaml
scene: !include scenes.yaml
`

Any help would be appreciated.

**Comment by PeterH24x7** [comment, marked as answer]
Hi Leigh - happy to help and sorry for your difficulties. It looks like your packages are defined under the "default_config" and not "home assistant". An extract of my configuration.yaml is as pictured. I hope this is useful.

![image](https://user-images.githubusercontent.com/91523850/208546378-bba72f2c-ca07-4671-9772-7a9c9ff9896e.png)

**Reply by PeterH24x7** [reply]
Also, you should need to do a full restart, rather once you've done a successful "check configuration" then simply reload the "rest entities and notify services" and "template entities". See marked image as follows.
<img width="449" alt="image" src="https://user-images.githubusercontent.com/91523850/208547214-d1c837f7-0fb6-4615-80ec-001da313f8cc.png">

**Reply by leighericksen** [reply]
Thanks Peter, 

Appreciate you taking the time to get back to me.

That got it all working - this integration is amazing and solves the issue I had without having to add another bit of equipment to the board. JUst need to add my other entities.

Thank you!
